md/two-main-types-of-deployment.md

```diff
@@ -50,7 +45,150 @@ But you can still use the pre packaged Tomcat bundles or the `bonita.war` webapp
 * more complex than the first deployment option (two application servers instead of one)
 * impact on performance (serialization + network overhead)
 
-### Accessible through RMI (EJB3)
+
+### Configuring the Bonita Stack
+
+The example below is for use with the Tomcat and Wildfly bundles. 
+
+Bonita Engine installation: follow the regular installation (see the [Tomcat](tomcat-bundle.md)
+or the [Wildfly](wildfly-bundle.md) bundle installation page) and use the setup tool
+to configure Bonita
+
+Bonita Portal installation
+* unarchive the binaries
+* configure custom authentication if required, see the [user authentication overview](user-authentication-overview.md) 
+* apply the following to make the Portal be a Client of the Bonita Engine
+* then you can start the Bonita Portal bundle with the startup script
+
+
+#### Bonita Portal 
+
+In this deployment, the Engine Client used by the Bonita Portal is configured by setting JVM System Properties for the following elements
+* instruct the Engine Client to use the HTTP protocol
+* set the url to the Engine
+* use username and password that match credentials configured for the platform admin (see the [Tomcat](tomcat-bundle.md)
+or the [Wildfly](wildfly-bundle.md) bundle configuration page)
+* for more details, please read [configuring the connection to a remote engine](configure-client-of-bonita-bpm-engine.md#client_config)
+
+In addition, all database datasources are disabled
+
+
+##### Common to all bundle types
+
+Update the `<bonita-installation-directory>/start-bonita.(sh|bat)` file to delete the following lines in order to only
+configure the bundle and bypass the Bonita initialization
+
+On Linux (start-bonita.sh)
+```bash
+  ./setup/setup.sh init $@
+  testReturnCode $?
+```
+
+On Windows (start-bonita.bat)
+```
+   call setup\setup.bat init %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
+   if errorlevel 1 (
+       goto exit
+   )
+```
+
```

Comment:
Should we add in this paragraph to remove the setup folder?

Reply:
I document to use the setup tool to configure the Bonita Portal bundle but for now this is only for the setenv file
I have to check how we document the alternative authentication configuration which is a mix between server side and Bonita configuration (which requires configuration in the setup tool only used with the Engine part here)

Reply:
done with an additional commit